app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -75,6 +78,20 @@ protected void onCreate(Bundle savedInstanceState) {
         }
         spinner_StartPos.setSelection(start_Pos_DropId);
 
+        // adds teams in match to the spinner
```

Comment:
Is this needed?  if we're on initial load of the page, Globals.CurrentMatchNumber will be empty, or -1 or something.  So this doesn't fill the spinner.  If it's a reload of the page (coming BACK from Match) then i think line 120-128 fills this spinner.  Am i missing something?